src/vivarium_census_prl_synth_pop/constants/data_values.py

```diff
@@ -77,6 +79,9 @@
 
 BUSINESS_MOVE_RATE_YEARLY = 0.1  # 10 changes per 100 py
 
+PERSONAL_INCOME_PROPENSITY_DISTRIBUTION = stats.norm(loc=0.0, scale=0.812309 ** 0.5)
+EMPLOYER_INCOME_PROPENSITY_DISTRIBUTION = stats.norm(loc=0.0, scale=0.187691 ** 0.5)
```

Comment:
Get rid of the spaces in the RHS of `scale`.

Reply:
Thankls